src/RestaurantDetail.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+export default function RestaurantDetail({ restaurantDetail }) {
+  if (!restaurantDetail) {
+    return <h3>텅~!</h3>;
+  }
+  const {
+    name, address, menuItems, reviews, information,
+  } = restaurantDetail;
+  return (
+    <div>
+      <h2>{name}</h2>
+      <div>
+        {'주소 : '}
+        <p>{address}</p>
+      </div>
```

Comment:
```suggestion
      <div>
        {`주소 : ${address}`}
      </div>
```

`template literal`로 처리해주는건 어떤가요? 의도와 다를까요?
- https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Template_literals

Reply:
address 만 test 하려고 하다 보니 구조분해 할당하면 getByText 안걸려서 따로 태그를 분리했었는데 
getByText(`주소 : ${address}`) 로 test 수정하고 template literal 적용했습니다!